shapedetector.py

```diff
@@ -1,5 +1,6 @@
-import cv2, imutils
-
+import cv2
+import imutils
+import numpy as np
 
 class ShapeDetector:
```

Comment:
what does this class do? you should provide a description of the class with comments

also give description to all functions too